.github/workflows/primer_run_main.yaml

```diff
@@ -96,7 +96,13 @@ jobs:
         run: |
           . venv/bin/activate
           pip install -e .
-          python tests/primer/primer_tool.py run --type=main
+          python tests/primer/primer_tool.py run --type=main 2>warnings.txt
+          WARNINGS=$(<warnings.txt)
+      - name: Emit Actions warnings for python warnings
```

Comment:
Since both of these steps are a normal `bash run`, can't they be merged?

Reply:
Just because I find this very neat:
https://stackoverflow.com/a/962268

Change you just made is fine with me though: explicit over intricate or whatever is in the PEP.

Reply:
Did you not merge these steps for a particular reason?